examples/routing_service/routing_service_with_security/routingServiceProfile.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0"?>
```

Comment:
Is there a strong reason for not having the encoding here?

``` xml
<?xml version="1.0" encoding="UTF-8"?>
```

@fgaranda ?

Reply:
Corrected this.